### ISSUE TITLE
nm only available with Xcode command line support

### ISSUE THREAD

**Issue by aonez** [aonez/NightShiftPatcher#1]
`nm` need to be installed. Calling it launches Apple installer.

**Comment by Squuiid** [comment]
Please could this be added to the script?
If you call nm right at the beginning of the script and pause for it to install then you could avoid the mess that is created for most users who don't have nm installed. 99% of users won't have Xcode tools installed.
Thank you for your efforts.

**Comment by aonez** [comment]
You had trouble with this? Maybe you're right, checking for it's existence and calling the installer before the rest of the script code might be the way to go. Anyway, worked for you?

**Comment by Squuiid** [comment]
Initially I ran it without having nm installed and it did not throw up an error. I think it should notify the user that Xcode tools must be installed and then quit the script.
Obviously Night Shift was not enabled in this case.
Once I installed Xcode tools I then reinstated the backup file from the desktop and re-ran the script. It worked great on my Mac Pro 5,1
Thanks again for your work and hopefully a few tweaks will make it the defacto Night Shift patch for macOS!
